.github/workflows/blackhole-post-commit.yaml

```diff
@@ -92,3 +92,17 @@ jobs:
   # to support the number of developers running this workflow
   # build-and-test-measure-perf:
   # build-and-test-measure-perf-device:
+
+  models-unit-tests:
+    needs: build-wheels
+    secrets: inherit
+    strategy:
+      fail-fast: false
+      matrix:
+        test-group: [
+          { arch: blackhole, runner-label: P100 },
+        ]
+    uses: ./.github/workflows/models-post-commit.yaml
```

Comment:
This needs to be removed now as Blackhole machines do not have access to Weka which is a requirement for this test.

If Weka is not a requirement, then this change can stay